pkg/querier/blocks_store_queryable.go

```diff
@@ -197,7 +198,10 @@ func NewBlocksStoreQueryable(
 }
 
 func NewBlocksStoreQueryableFromConfig(querierCfg Config, gatewayCfg storegateway.Config, storageCfg mimir_tsdb.BlocksStorageConfig, limits BlocksStoreLimits, logger log.Logger, reg prometheus.Registerer) (*BlocksStoreQueryable, error) {
-	var stores BlocksStoreSet
+	var (
+		stores       BlocksStoreSet
+		bucketClient objstore.Bucket
```

Comment:
Note to reviewers: define `bucketClient objstore.Bucket` because now `bucket.NewClient()` returns a `objstore.InstrumentedBucket`.